### ISSUE TITLE
(Repeater) Missing a specify post number in Query - count (Oxygen 3.1. Beta 1)

### ISSUE THREAD

**Issue by markoss20** [soflyy/oxygen-bugs-and-features#657]
Before: 
![image](https://user-images.githubusercontent.com/7726041/67689761-13911380-f99c-11e9-8647-708f75a15678.png)

In Oxygen 3.1 Beta 1:
![image](https://user-images.githubusercontent.com/7726041/67689820-31f70f00-f99c-11e9-9c31-0bde0590926e.png)

In a repeater block is always pagination. When you need just 3 posts. You need to hide pagination thought css.

Wordpress 5.2.4
Oxygen 3.1. Beta 1
Chrome 78

**Comment by nickfmc** [comment]
well posts per page is more accurate to how the WP query works, the query would have been calling this previously.  I haven't played with the post repeater much, sounds to me like they just added pagination to the query?  Your may want to adjust your request to be "Unable to disable pagination on repeater helper"

**Comment by markoss20** [comment]
> well posts per page is more accurate to how the WP query works, the query would have been calling this previously. I haven't played with the post repeater much, sounds to me like they just added pagination to the query? Your may want to adjust your request to be "Unable to disable pagination on repeater helper"

Yes, maybe this is the solution. Or just a simple field: "or specify the number."
